tfhe/docs/fine_grained_api/shortint/operations.md

```diff
@@ -72,8 +72,8 @@ fn main() {
     server_key.unchecked_mul_lsb_assign(&mut ct_1, &ct_2);
 
     // We use the client key to decrypt the output of the circuit:
-    let output = client_key.decrypt(&ct_1);
-    println!("expected {}, found {}", ((msg1 * scalar as u64 - msg2) * msg2) % modulus as u64, output);
+    let output = client_key.decrypt_decode_padding(&ct_1);
```

Comment:
I think we should keep the simple `decrypt` that returns the msg as we previously did, as I think this is what users (and us) care about most of the time

Reply:
agreed for most of our code, haven't had time to review all this yet